bedrock/newsletter/templates/newsletter/updated.html

```diff
@@ -58,56 +56,56 @@ <h4>{{_('Would you mind telling us why you’re leaving?') }}</h4>
       <div class="mzp-l-content mzp-t-content-md">
         <section class="c-updated-block">
           <header class="c-updated-block-header">
-            <h1>{{ _('You’ve been unsubscribed.') }}</h1>
-            <h2>{{ _('We’re sorry to see you go.')}}</h2>
-            <p>{{ _('Thanks for telling us why you’re leaving.') }}</p>
+            <h1>{{ ftl('newsletters-youve-been-unsubscribed') }}</h1>
+            <h2>{{ ftl('newsletters-were-sorry-to-see-you-go') }}</h2>
+            <p>{{ ftl('newsletters-thanks-for-telling-us-why') }}</p>
           </header>
         </section>
       </div>
     {% else %}
       <div class="mzp-l-content">
         <section class="c-updated-block">
-          {% if l10n_has_tag('newsletter_confirm_0320') %}
```

Comment:
Looks like this conditional copy was never exposed for L10n, so changing it to hard-coded here for now.